<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>26</number>
     </property>
     <property name="topMargin">
      <number>26</number>
     </property>
     <property name="rightMargin">
      <number>26</number>
     </property>
     <property name="bottomMargin">
      <number>26</number>
     </property>
     <property name="spacing">
      <number>16</number>
     </property>
     <item row="0" column="0">
      <widget class="LargeTitleLabel" name="VideoTitile">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>视频</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="File">
       <item>
        <widget class="SubtitleLabel" name="FileTitle">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SimpleCardWidget" name="FileBox">
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="BodyLabel" name="inputLabel">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>视频输入</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="LineEdit" name="InputLine">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PushButton" name="InputButton">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="BodyLabel" name="OutputLabel">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>视频输出</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="LineEdit" name="OutputLine">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PushButton" name="Outputbutton">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="BodyLabel" name="TextLabel">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>字幕</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="LineEdit" name="TextLine">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="PushButton" name="Outputbutton_2">
              <property name="text">
               <string>浏览</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="Encode">
       <item>
        <widget class="SubtitleLabel" name="FileTitle_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>编码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SimpleCardWidget" name="SimpleCardWidget">
         <layout class="QGridLayout" name="gridLayout_9">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>16</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="BodyLabel" name="BodyLabel">
                  <property name="text">
                   <string>编码器</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="ComboBox" name="ComboBox">
                  <property name="text">
                   <string>x264</string>
                  </property>
                  <property name="checkable">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="BodyLabel" name="BodyLabel_3">
                  <property name="text">
                   <string>位深度</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="ComboBox" name="ComboBox_2">
                  <property name="text">
                   <string>8bit</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>16</number>
                </property>
                <item>
                 <widget class="BodyLabel" name="BodyLabel_2">
                  <property name="text">
                   <string>起始帧</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBox" name="SpinBox"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                 <number>16</number>
                </property>
                <item>
                 <widget class="BodyLabel" name="BodyLabel_4">
                  <property name="text">
                   <string>编码帧数</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="SpinBox" name="SpinBox_2"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="parms">
       <item>
        <widget class="SubtitleLabel" name="FileTitle_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SimpleCardWidget" name="SimpleCardWidget_2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>36</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="RadioButton" name="RadioButton">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>CRF(固定码率因子)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="RadioButton" name="RadioButton_2">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>VBR(可变码率控制)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="RadioButton" name="RadioButton_3">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>2Pass</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_10">
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>35</number>
            </property>
            <property name="spacing">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="BodyLabel" name="BodyLabel_5">
              <property name="text">
               <string>CRF</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="DoubleSpinBox" name="DoubleSpinBox"/>
            </item>
            <item row="1" column="1">
             <widget class="BodyLabel" name="BodyLabel_6">
              <property name="text">
               <string>Kbps</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="SubtitleLabel" name="FileTitle_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>分辨率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SimpleCardWidget" name="SimpleCardWidget_3">
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <widget class="SwitchButton" name="SwitchButton">
              <property name="text">
               <string>保持原分辨率</string>
              </property>
              <property name="onText">
               <string>自定义分辨率</string>
              </property>
              <property name="offText">
               <string>保持原分辨率</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="BodyLabel" name="BodyLabel_7">
                <property name="text">
                 <string>宽度</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SpinBox" name="SpinBox_3"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="BodyLabel" name="BodyLabel_8">
                <property name="text">
                 <string>高度</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="SpinBox" name="SpinBox_4"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="SubtitleLabel" name="FileTitle_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>压制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SimpleCardWidget" name="SimpleCardWidget_4">
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>36</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>36</number>
          </property>
          <property name="verticalSpacing">
           <number>25</number>
          </property>
          <item row="0" column="0">
           <widget class="CheckBox" name="CheckBox">
            <property name="text">
             <string>完成后自动关机</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="PrimaryPushButton" name="PrimaryPushButton">
            <property name="text">
             <string>开始压制</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LargeTitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
